src/main/python/gui/modulespecification_dialog.py

```diff
@@ -302,18 +325,18 @@ def validate_and_save(self, addanother=False, closedialog=False):
         modulevalid, modulemessage = self.module_widget.validity_check()
 
         messagestring = ""
-        if not (articulatorsvalid and timingvalid):
+        if not articulatorsvalid:
```

Comment:
ok so this was because I originally thought I needed to single out `articulatorsvalid`...

However, I think what I have now is easier to read as it has less conditionals & I've tested it still gives the same error messages.
But if we prefer the old way, I'm completely fine to change it

Reply:
Seems reasonable to me :)